oak/server/oak_node.cc

```diff
@@ -167,42 +167,48 @@ std::string Sha256Hash(const std::string& data) {
 }
 
 OakNode::OakNode(const std::string& node_id, const std::string& module)
-    : Service(), node_id_(node_id), module_hash_sha_256_(Sha256Hash(module)) {
+    : Service(), node_id_(node_id), module_hash_sha_256_(Sha256Hash(module)) {}
+
+std::unique_ptr<OakNode> OakNode::Create(const std::string& node_id, const std::string& module) {
   LOG(INFO) << "Creating Oak Node";
-  wabt::Result result;
-  InitEnvironment(&env_);
-  LOG(INFO) << "Func count: " << env_.GetFuncCount();
+
+  OakNode* node = new OakNode(node_id, module);
```

Comment:
Is it worth wrapping this here rather than delaying it?

Reply:
Sorry, not quite sure what you mean?

Reply:
`std::unique_ptr<OakNode> node = absl::WrapUnique(new OakNode(node_id, module));`
 
If we follow this pattern it's easy to spot cases in which we forget to wrap the `new`s.

Reply:
Ah, got you now -- done.